components/bl-carousel-component/src/subcomponents.js

```diff
@@ -98,24 +98,24 @@ export function CarouselIndicators(props) {
   );
 }
 
-export function NextButton({ disabled, onNextButton }) {
+export function NextButton({ disabled, onNextButtonClick }) {
```

Comment:
As we already here. Please, refactor that to one component Button which accept type and icon component as props.